DevelopingSpace/starchart
# Fix multiple notifications at once bug

## Description 
Fixes #650 by making adjustments to expiration notification-related code for DNS records and Certificates. Attempts to fix the annoying bug of getting too many notifications at once regarding expiring records. _Still need to work on the bug that sends multiple notifications when a certificate fails.

It seems like the root cause of getting multiple notifications was because we were checking if `lastNotified` is set to `null` OR `lastNotified` is set to date, which is 30 days in the past. Since we are sending one reminder for now, we don't need to verify whether it has been a month since we last notified the user.

### Steps to test
1. Run the project locally
2. Request a certificate or DNS record
3. Run `npm run db:studio` and set the expiration date to _a month_ in the future
4. Navigate to http://localhost:8025 to see a single notification after some time:
<img width="1440" alt="demo" src="https://user-images.githubusercontent.com/50856799/234404237-34728467-edea-4c81-b28e-ec1a2a2f96a6.png">
